spec/javascripts/item_selector_limit_spec.js

```diff
@@ -37,16 +37,15 @@ describe('Item Selector', () => {
       itemSelectorLimit.checkboxes().filter(':nth-child(3)').trigger('click');
       expect(itemSelectorLimit.checkboxes().filter(':checked').length).toBe(3);
 
-      var fourthCheckbox = itemSelectorLimit.checkboxes()
+      const fourthCheckbox = itemSelectorLimit.checkboxes()
                                             .filter(':nth-child(4)');
-      var spy = jest.spyOn(fourthCheckbox, 'click');
-      fourthCheckbox.click();
-      expect(spy).toHaveBeenCalled();
```

Comment:
This spy didn't make much sense. It's just checking that we called click on line 43.